README.md

```diff
@@ -248,9 +248,17 @@ If you want to run the Snapshotter Lite Node without Docker, you need to make su
 
 5. Your node should start in background and you should start seeing logs in your terminal.
 6. To stop the node, you can run `pkill -f snapshotter` in a new terminal window.
-  
+
 ## Monitoring and Debugging
 
+### Monitoring
+
+To enable Telegram reporting for snapshotter issues:
+1. Search for `@PowerloomReportingBot` in the Telegram App and start a conversation.
+2. Start the bot by typing the `/start` command in the chat. You will receive a response containing your `Chat ID` for the bot.
+3. Enter the `Chat ID` when prompted on node startup.
```

Comment:
@Seth-Schmidt this isn't prompted and they need to manually add to their `.env` right?

Reply:
The new prompt should be included in this PR:

https://github.com/PowerLoom/snapshotter-lite-v2/blob/b41abf6f3b8ebb4e13e3269ecfe24911c1924a43/build-dev.sh#L38